src/Cake.Common/Tools/OctopusDeploy/OctopusDeployRunner.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using Cake.Core;
+using Cake.Core.IO;
+using Cake.Core.Utilities;
+
+namespace Cake.Common.Tools.OctopusDeploy
+{
+    /// <summary>
+    /// The OctopusDeploy runner
+    /// </summary>
+    public sealed class OctopusDeployRunner : Tool<CreateReleaseSettings>
```

Comment:
With the way Runners are structured, they're intended to be tightly coupled to a single set of settings. So you're have a create release runner, and a deploy runner, all which are bootstrapped by aliases. Correct me if I'm wrong @patriksvensson .